app/buyers/views/buyers.py

```diff
@@ -302,8 +300,8 @@ def view_brief_responses(framework_slug, lot_slug, brief_id):
     brief_responses = data_api_client.find_brief_responses(brief_id)['briefResponses']
 
     brief_responses_require_evidence = (
-        datetime.strptime(current_app.config['FEATURE_FLAGS_NEW_SUPPLIER_FLOW'], "%Y-%m-%d")
-        <= datetime.strptime(brief['publishedAt'][0:10], "%Y-%m-%d")
+        datetime.strptime(current_app.config['FEATURE_FLAGS_NEW_SUPPLIER_FLOW'], "%Y-%m-%d") <=
```

Comment:
I don't really like this convention of having the operator on the first line; I think it looks better and flows better if it's on the following line. Looking at this specific example, it seems to be far easier to look at the comparison in the 'old' style and see all three bits of information you need in one place (the left-hand side): datetime, <=, datetime. In the new style, the operator is out of the way and you have to switch looking from left to right to left to read it all. It's easier to make a mistake, skip a line, forget the operator, etc. IMO.